kubernetes/chart/zulip/Chart.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: v2
+description: zulipchat
+name: zulip
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.2.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "4.7-0"
```

The GH release page only states `4.7`

```suggestion
appVersion: "4.7"
```

---

The [Docker tag I'm installing is `4.7-0`](https://github.com/greenhost/docker-zulip#welcome-to-docker-zulip), so I prefer to keep it like this to make that clear.

---

What about this change ? I don't see it in your last 2 commits

---

OK!